src/user/supla_esp.h

```diff
@@ -191,7 +191,8 @@ extern const uint8_t rsa_public_key_bytes[RSA_NUM_BYTES];
 	PIN_FUNC_SELECT(PERIPHS_IO_MUX_GPIO5_U, FUNC_GPIO5); \
 	PIN_FUNC_SELECT(PERIPHS_IO_MUX_MTDI_U, FUNC_GPIO12); \
 	PIN_FUNC_SELECT(PERIPHS_IO_MUX_MTCK_U, FUNC_GPIO13); \
-	PIN_FUNC_SELECT(PERIPHS_IO_MUX_MTMS_U, FUNC_GPIO14)
+	PIN_FUNC_SELECT(PERIPHS_IO_MUX_MTMS_U, FUNC_GPIO14); \
```

Comment:
Do not modify files in the "user" directory.
As you can see there is a #ifndef macro there. Declare GPIO_PORT_INIT in sonoff_pow_R2.h